src/bootstrap/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 None.
 
+## [Version 3] - 2020-10-03
```

Comment:
I don't think this is worth a major change -- it will build strictly less than before, right? People don't need to be notified of changes that likely don't affect their workflow. The host/build change was much more likely to break your build in a confusing way, here you're just going to get less built and can easily opt back in.